pkg/security/probe/probe_windows.go

```diff
@@ -236,24 +273,30 @@ func (p *WindowsProbe) setupEtw(ecb etwCallback) error {
 						delete(filePathResolver, ca.fileObject)
 					}
 				}
+				p.stats.fileClose++
 			case idFlush:
 				if fa, err := parseFlushArgs(e); err == nil {
 					ecb(fa, e.EventHeader.ProcessID)
 				}
+				p.stats.fileFlush++
+
+			// cases below here are file id events we're not currently processing.
+			// for now, just count them so that we can get an idea of frequency/voluem
 			case idSetInformation:
-				fallthrough
+				p.stats.fileSetInformation++
+
 			case idSetDelete:
-				fallthrough
+				p.stats.fileSetDelete++
+
 			case idRename:
-				fallthrough
+				p.stats.fileidRename++
 			case idQueryInformation:
-				fallthrough
+				p.stats.fileidQueryInformation++
 			case idFSCTL:
-				fallthrough
+				p.stats.fileidFSCTL++
+
 			case idRename29:
-				if sia, err := parseInformationArgs(e); err == nil {
```

Comment:
I think if we remove the call to this parsing call, we should remove the function as well (cf comment earlier)

Reply:
the parsing function is coming back for the next release.

Reply:
oh ok !